webapp/src/main/resources/public/js/ict/IctModalContainer.js

```diff
@@ -55,7 +55,7 @@ class IctModalContainer extends React.Component {
             'repoName': this.state.selectedTextUnit.repositoryName,
             'locales': this.state.selectedTextUnit.locale,
             'searchText': this.state.selectedTextUnit.textUnitName,
-            'asset_path': this.state.selectedTextUnit.asset_path
+            'asset_path': this.state.selectedTextUnit.assetName
```

Comment:
Actually we should update the query parameter too for consistency and change the analytics app